src/db_api_util.ts

```diff
@@ -677,6 +680,24 @@ export async function setThreadNeedsAttentionToDb(
   }
 }
 
+export async function setThreadRoutedToDb(
```

Comment:
Does `routed` mean routed to a state channel? if so can you rename?

Reply:
It's more like `isThreadRouted` but that `ToDb` suffix is used on a bunch of methods... seems redundant tho given we're in DbApiUtil namespace